docs/reference/errors-and-warnings/NU3026.md

```diff
@@ -0,0 +1,24 @@
+---
+title: NuGet Warning NU3026
+description: NU3026 Warning code
+author: heng-liu
+ms.author: henli
+ms.date: 3/18/2019
+ms.topic: reference
+ms.reviewer: dtivel
+f1_keywords: 
+  - NU3026
+---
+
+# NuGet Warning NU3026
+
+<pre>Package 'SamplePackage v1.0.0' from source 'http://contoso.com/index.json': The timestamp response is invalid. Nonces did not match.</pre>
+
+### Issue
+
+The timestamp authority did not return an expected nonce in its response; therefore, its response is invalid.
```

Comment:
I have no idea what is a "nonce". Is that a term that nuget package consumers are expected to know? If not, can we simplify the message?
cc: @rido-min